File: 1 - Iniciante/C99/BEE1064.c
#include <stdio.h>

int main(void){
	int i, pos=0;
	float num, soma=0;
	
	for (i=1; i<=6; i++){
		scanf("%f", &num);
		
		if (num>0){
			pos += 1;
			soma += num;
		}
	}
	
	printf("%d valores positivos\n", pos);
	printf("%.1f\n", soma/pos);
	
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1064.py
vec = []
ctd = 0
soma = 0
for i in range(6):
    vec.append(float(input()))
    if (vec[i]>0):
        soma += vec[i]
        ctd += 1 

print(f"{ctd} valores positivos")
print(f"{soma/ctd:.1f}")